.aicoe-ci.yaml

```diff
@@ -0,0 +1,5 @@
+check:
+  - thoth-tox
```

Comment:
Which Python interpreter version will run this?

Reply:
we are currently using this image: docker.io/frenzymadness/fedora-python-tox:latest
which 3.5,3.6,3.7,3.8,3.9 python versions.

Reply:
Please, switch to https://hub.docker.com/r/fedorapython/fedora-python-tox/ - it's the same image but newly under an organization instead of my personal account.

Reply:
Also, the new version contains Python 3.10.

Reply:
surely , thanks for updating :)